ansible/roles/debops.discourse/tasks/main.yml

```diff
@@ -0,0 +1,79 @@
+---
+
+- name: Install required APT packages
+  apt:
+    name: "{{ query('flattened', [
+      discourse__base_packages,
+      discourse__release_packages[discourse__distribution_release] | d([]) ,
+      discourse__database_packages[discourse__distribution_release][discourse__database] | d([])
+       ] ) }}"
+    state: 'present'
+    install_recommends: False
+  register: discourse__register_packages
+  until: discourse__register_packages is succeeded
+
+- name: Install required node.js packages
+  npm:
+    name: '{{ item.key }}'
+    version: '{{ item.value or omit }}'
+    global: yes
+  with_dict: '{{ discourse__nodejs_packages }}'
+  register: discourse__register_nodes_packages
+  until: discourse__register_nodes_packages is succeeded
```

Comment:
I wonder if the `node` packages could be installed on the `discourse` UNIX account instead. Would that work as expected?